src/SkillRunner/bot/room_details.py

```diff
@@ -0,0 +1,73 @@
+from .mention import Mention
+
+# TODO: Is there a Python equivalent to C#'s TimeSpan?
+
+class Threshold(object):
+    """
+    Represents a warning and deadline threshold.
+    """
+    def __init__(self, warning, deadline):
```

Comment:
I've been trying to include typing hints. Python has [a built-in set of TypeScript-style type annotations](https://docs.python.org/3/library/typing.html).

So for example, combining this with @dahlbyk 's suggestion, this would be:

```suggestion
    def __init__(self, warning: timedelta, deadline: timedelta):
```